src/console/CmdProcessor.cpp

```diff
@@ -336,12 +336,16 @@ void CmdProcessor::processServerCmd(folly::StringPiece cmd) {
         if (spaceName && !spaceName->empty()) {
             curSpaceName_ = std::move(*spaceName);
         }
-        printResult(resp);
-        if (resp.get_rows() != nullptr) {
+        if (resp.get_rows() && !resp.get_rows()->empty()) {
+            printResult(resp);
             std::cout << "Got " << resp.get_rows()->size()
                       << " rows (Time spent: "
                       << resp.get_latency_in_us() << "/"
                       << dur.elapsedInUSec() << " us)\n";
+        } else if (resp.get_rows() && resp.get_rows()->empty()) {
```

Comment:
`resp.get_rows()->empty()` here is always true if `resp.get_rows() != nullptr`

Reply:
true?   false ?

Reply:
The result displays as follow:
```
(user@127.0.0.1) [myspace]> show tags;
==========
| Name   |
==========
| player |
----------
Got 1 rows (Time spent: 1185/1733 us)

(user@127.0.0.1) [myspace]> show edges;
Empty set (Time spent: 1740/2362 us)
```

I had used gdb  to debug as follows: 

In nebula::graph::CmdProcessor::processServerCmd  function, use show edges as above:

```
(gdb) p	resp.get_rows()
$1 = (std::vector<nebula::graph::cpp2::RowValue, std::allocator<nebula::graph::cpp2::RowValue> > *) 0x7ffe68672250
(gdb) p	resp.get_rows()->empty()
$2 = true
```

Reply:
I meant the test `(resp.get_rows() && resp.get_rows()->empty())` is identical to `(resp.get_rows() != nullptr)`

Reply:
I don't think so.
 case:
```
(user@127.0.0.1) [myspace]> show tags;
==========
| Name   |
==========
| player |
----------
Got 1 rows (Time spent: 1185/1733 us)

(user@127.0.0.1) [myspace]> show edges;
Empty set (Time spent: 1740/2362 us)
```

when `show tags`,  execute the branch `(resp.get_rows() && !resp.get_rows()->empty())`
when `show edges`,  execute the branch `(resp.get_rows() && resp.get_rows()->empty())`

Reply:
I think you did not quite understand me.

Reply:
when `resp.get_rows() != nullptr `,   the value of` resp.get_rows()->empty()` is true or false.
In above case, execute` show tags`, resp.get_rows()->empty()  is false
execute` show edges`, `resp.get_rows()->empty()`  is true

Reply:
Upon reaching the `else if ()` branch, the `if ()` branch was not been chosen, right? 

Thus `resp.get_rows() == nullptr` or `resp.get_rows().empty() == true`, right?

Thus in the `else if ()` branch, if `resp.get_rows() != nullptr`, then `resp.get_rows().empty() == true`, right?

Reply:
Yeah.